common/common_helper.sh

```diff
@@ -373,3 +373,21 @@ function check_kind() {
 		exit 1
 	fi
 }
+
+###########################################
+#   Delete namespace
+###########################################
+function delete_namespace() {
+  DAPP_NAMESPACE=$1
+  echo
+  echo "#########################################"
+  # Check if the namespace exists
+    if kubectl get namespace "${DAPP_NAMESPACE}" > /dev/null 2>&1; then
```

Comment:
@shreyabiradar07 - Is this working? It didn't delete the namespace for me.

Reply:
on terminating local monitoring script, it deletes `test-mulitple-import` namespace (if the namespace is created successfully while running the demo script)

```
./local_monitoring_demo.sh -c openshift -i quay.io/kruize/autotune:hackdemo -t
 
1. Deleting autotune git repos
 
#########################################
 Deleting namespace: test-multiple-import
 namespace "test-multiple-import" deleted
#########################################
 
Success! Kruize demo cleanup took 15 seconds
```